clients/src/test/java/org/apache/kafka/common/security/authenticator/SaslAuthenticatorFailureDelayTest.java

```diff
@@ -228,12 +213,8 @@ private NioEchoServer createEchoServer(SecurityProtocol securityProtocol) throws
     }
 
     private NioEchoServer createEchoServer(ListenerName listenerName, SecurityProtocol securityProtocol) throws Exception {
-        if (failedAuthenticationDelayMs != -1)
```

Comment:
The two parameters we use in the test are `0` and `200`, so this is dead code.

Reply:
It's not dead code, it sets the delay in Selector level.